### ISSUE TITLE
Issue with loading HCP phase data from EDAX .ang files

### ISSUE THREAD

**Issue by gsparks3** [pyxem/orix#534]
Attempting to load an EDAX .ang file containing a HCP phase (e.g. alpha titanium) using orix.io.load() fails. The core issue seems to be that EDAX's phase definition is using "62" as an alias for the actual symmetry point group, similar to how they use "43" for cubic systems, but the alias is not recognized. 

Attached is a small Ti EBSD .ang file which reproduces the issue. 

[Ti_single_crystal_scan.zip](https://github.com/user-attachments/files/18115333/Ti_single_crystal_scan.zip)

**Comment by hakonanes** [comment]
Thanks for the report and the dataset, @gsparks3.

I see EMsoft has a complete (?) list [here](https://github.com/EMsoft-org/EMsoftOO/blob/develop/Source/EMsoftOOLib/mod_symmetry.f90#L565):

```fortran
character(2), public, dimension(32) :: TSLsymtype = (/' 1',' 1',' 2',' 2',' 2','22','22','22', &
                                                       ' 4',' 4',' 4','42','42','42','42',' 3', &
                                                       ' 3','32','32','32',' 6',' 6',' 6','62', &
                                                       '62','62','62','23','23','43','43','43'/)
```

I'll add the same mapping to our list.

**Comment by hakonanes** [comment]
@gsparks3, newly released 0.13.3 has this fixed.

**Comment by gsparks3** [comment]
Excellent, thanks!
